templates/base/footer.tmpl

```diff
@@ -89,6 +89,37 @@
 		issuesTribute.attach(document.getElementById('content'))
 	</script>
 	{{end}}
+	<script>
```

Comment:
Is there a reason to not put this in `/public/js/index.js`?

Reply:
@kolaente it can't be put (at least currently) in index.js as it has server-side condition check if this should be included so such refactoring would probably be better done in different PR

Reply:
I tried putting it to `/public/js/index.js`, and limited the initialization of tribute to a condition, wether an element with id `content` is available.

But I think it is much cleaner, if the emojiTribute is only initialized when the page requiring it is loaded.

Reply:
oh ok I see...